admin/views/check-form/upload-new-forms.ejs

```diff
@@ -0,0 +1,122 @@
+<% const err = error %>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-third">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds check-forms">
+
+        <% if (messages && messages.error && messages.error.length) { %>
+        <div class="error-summary" role="group" aria-labelledby="error-summary-heading-1">
+            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-1">
+                There is a problem with the file name
+            </h2>
+            <p><%= messages.error %></p>
+            <ul class="error-summary-list">
+                <li><a href="#file-upload">Please try uploading a file again</a></li>
+            </ul>
+        </div>
+        <% } %>
+
+        <% if (error) { %>
+        <% if (error.hasOwnProperty('message')) { %>
+        <div class="error-summary" role="group" aria-labelledby="error-summary-heading-2">
+            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-2">
+                <%= error.message %>
+            </h2>
+
+            <p>The form upload spreadsheet:</p>
+
+            <ul class="list list-bullet">
+                <li> must be in CSV format - <a href="#csv-guidance">see guidance for instructions</a></li>
+                <li> must only contain 2 columns</li>
+                <li> must not have a header row</li>
+                <li> must only contain numbers between 1 and 12</li>
+                <li> must contain exactly 25 rows of data</li>
+            </ul>
+
+            <ul class="error-summary-list">
+                <li><a href="#file-upload">Please try uploading a file again</a></li>
+            </ul>
+        </div>
+        <% } %>
+        <% } %>
+
+        <h1 class="heading-xlarge">Upload new form</h1>
+
+        <%- partial('partials/_session_expiry') %>
+
+        <h2 class="heading-medium top-padding-30"><a name="csv-guidance">Download example</a></h2>
+
+        <ol class="numeric-list">
+            <li>
+                Download the example
+                <div class="panel panel-border-wide short">
+                    <a href="<%= assetPath %>csv/check-form-sample.csv" class="heading-small icon-download">Check form example (CSV, 4KB)</a>
+                </div>
+            </li>
+            <li>Open file in a programme like Excel, OpenOffice or Google Sheets</li>
```

Comment:
This may be the actual business wording, but how many of our users would call Excel a 'programme'? 👀

Reply:
@jon-shipley 😆

Reply:
jokes aside - probably best to report this to the BAs

Reply:
@GuyHarwood reported already and I was told there is a content editor coming in to update the content.